src/main/java/com/teamknp/hotel/controller/LoginController.java

```diff
@@ -0,0 +1,28 @@
+package com.teamknp.hotel.controller;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class LoginController {
+
+        @GetMapping("/")
+        public String home1() {
```

Comment:
Ojoj, home1?
Nie możemy mieć takiej nazwy! Skoro home1 i home to praktycznie to samo, to może by tak po prostu zmodyfikować mapping?